src/test/java/com/algorand/algosdk/integration/Stepdefs.java

```diff
@@ -158,6 +158,59 @@ public SignedTransaction signWithAddress(Transaction tx, Address addr) throws co
         return acct.signTransaction(tx);
     }
 
+    /**
+     * advanceRound is a convenience method intended for testing with DevMode networks.
+     * <p>
+     * Since DevMode block generation requires a transaction rather than time passing, test assertions may require advancing rounds.  advanceRound issues advanceCount payments to advance rounds.
+     */
+    private void advanceRoundsV1(int advanceCount) {
```

Comment:
If we always advance one round, do we need a parameter here?

Alternatively, if we want to send multiple transactions, it might be good to randomize the amount field or the note field as duplicate transaction might fail.

Reply:
@algochoi Sorry - I meant to add randomization and forgot to circle back.  I updated via https://github.com/algorand/java-algorand-sdk/pull/350/commits/d0aa73332b7e81035892c81494edffa48e12c9af.  Let me know what you think?

Reply:
Looks good! Although if you think we would advance a lot of blocks in the future, a bigger range of random numbers would be better

Reply:
@algochoi How does https://github.com/algorand/java-algorand-sdk/pull/350/commits/3390a6d687c5e57c349064de5014afc4f34a3b2d look to you?

Additionally, I limited the upperbound proportional to the funding amount.